marklogic-data-hub-central/ui/src/components/login-form/login-form.tsx

```diff
@@ -83,7 +84,7 @@ const LoginForm: React.FC = () => {
           >
             <Input
               id="username"
-              prefix={<Icon type="user" className={styles.usernameIcon} />}
+              prefix={<Person className={styles.usernameIcon} />}
```

Comment:
In login-form.module.scss, let's adjust the CSS for these icons:
```
  .usernameIcon, .passwordIcon {
    color: rgba(0,0,0,.25);
    margin-left: -2px;
  }

  .usernameIcon {
    font-size: 17px;
  }

  .passwordIcon {
    font-size: 15px;
  }
```
Also, remove the CSS for .hostIcon (two places) as that is no longer used.